models/model.js

```diff
@@ -43,5 +44,17 @@ const readArticles = () => {
 };
 
 
-
-module.exports = {readTopics,readArticles,readArticleById}
+const createComment = (article_id, username, body) => {
+  if (!Number.isInteger(Number(article_id))) {
```

Comment:
We don't need to explicitly check for this - psql will send an error that you can catch.